fgpyo/alignment/__init__.py

```diff
@@ -0,0 +1,232 @@
+from typing import Callable
```

Comment:
I'd move this to `fgpyo/alignment.py` and then move the tests similarly.  Any reason to keep it this way?

Reply:
Don't forget to add module docs.  Can you also add an example or two of the padded string method?  See examples in other modules: https://github.com/fulcrumgenomics/fgpyo/blob/5ef0f5e9ced1e8e3763e6858317b59e448193635/fgpyo/read_structure.py#L16-L49

```suggestion
"""
Classes for representing Alignments
-----------------------------------

<INSERT an example or TWO HERE>

Module Contents
~~~~~~~~~~~~~~~
The module contains the following public classes:
    - :class:`~fgpyo.alignment.Alignment` -- Describes the alignment between two sequences 

"""


from typing import Callable
```